### PR TITLE
A compromise PR of a long original work

### PR DESCRIPTION
**Pull request by sdake**
See PR: https://github.com/istio/istio.io/pull/5142

Pretty much everything about this PR is compromised...

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[X] Docs
[X] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by sdake** [comment]
rendering: https://deploy-preview-5146--preliminary-istio.netlify.com/docs/setup/install/standalone-operator/

File: content/en/docs/setup/install/standalone-operator/index.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
+---
+title: Standalone Operator Quick Start Evaluation Install
```
Comment (howardjohn) [inline content/en/docs/setup/install/standalone-operator/index.md]:
```suggestion
title: Standalone Operator Quick Start Evaluation Install [Experimental]
```

To match the istioctl cli docs.

Comment (sdake) [reply content/en/docs/setup/install/standalone-operator/index.md]: done.

File: content/en/docs/setup/install/standalone-operator/index.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+required are a supported Kubernetes cluster and the `kubectl` command. This
+installation method lets you quickly evaluate Istio in a Kubernetes cluster on
+any platform using a variety of profiles.
+
+To install Istio for production use, we recommend using the [Helm Installation guide](/docs/setup/install/helm/)
+instead, which is a stable feature.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/#downloading-the-release).
```
Comment (howardjohn) [inline content/en/docs/setup/install/standalone-operator/index.md]: no longer needed, right? or should we still encourage it to get istioctl

File: content/en/docs/setup/install/standalone-operator/index.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Verifying the installation
+
+{{< warning >}}
+This document is a work in progress. Expect verification steps for each of the profiles to
```
Comment (howardjohn) [inline content/en/docs/setup/install/standalone-operator/index.md]: make sure we remove this before 1.4 ships

File: content/en/docs/setup/install/standalone-operator/index.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    {{< /text >}}
+
+## Deploy your application
+
+You can now deploy your own application or one of the sample applications
+provided with the installation like [Bookinfo](/docs/examples/bookinfo/).
+
+{{< warning >}}
+The application must use either the HTTP/1.1 or HTTP/2.0 protocols for all its HTTP
+traffic; HTTP/1.0 is not supported.
```
Comment (howardjohn) [inline content/en/docs/setup/install/standalone-operator/index.md]: is this from the current eval page? If doesn't seem too relevant but if it's already there seems find the keep it

File: content/en/docs/setup/install/standalone-operator/index.md (hunk 3 above)
Comment (sdake) [reply content/en/docs/setup/install/standalone-operator/index.md]: yes once we have examples of verirfication they will be integrated in the docs.  from my understanding profile switching just became functional a few hours ago.

File: content/en/docs/setup/install/standalone-operator/index.md (hunk 4 above)
Comment (sdake) [reply content/en/docs/setup/install/standalone-operator/index.md]: i copied this page from the current eval page - then modified to suit the standalone operator.

File: content/en/docs/setup/install/standalone-operator/index.md (hunk 2 above)
Comment (sdake) [reply content/en/docs/setup/install/standalone-operator/index.md]: You may be correct removing may be the best option.  We do want people to download istioctl, but everytime someone wants me to install a downloaded binary in my path, I do this:

https://images.genius.com/2e17391a56a8c0441686b577da3678c9.500x376x35.gif

IOW the overhead of installing a binary slows people down from evaluating Istio.  Add to that tar.  I know these seem like trivial things, but the way these projects typically work out is a CTO or CIO office is asked to evaluate a project.  If they have do these "standard magic incantations" to get a working system, they have a negative impact on the resulting white paper.  I know for certain - I've written several while working in a CTO office 👍

**Comment by sdake** [comment]
> overall looks good
> 
> I'm not sure it's the best move to copy the operator yamls here, but I don't have that strong of an opinion

Ya, I wasn't super keen on that approach either - however it was requested.  I can move these to some other location such as GCS if we can get the redirect to work and you can provide some pointers to uploadinng these to gcs (or upload them yourself).

**Pull request merged**